scripts/test-sim-dev.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+CONTAINER_NAME=storj_sim_postgres
+docker run -d --rm -p 5433:5432 --name $CONTAINER_NAME postgres
```

Comment:
I think that we should use the same Postgres version than the one used in production.
I don't know which one we currently use, but defaulting to `latest` docker image tag isn't good because at some point (if not currently) it may pull a different postgres version than the one used in production.

Reply:
good point

@stefanbenten @brimstone what version of postgres is used (or will be used soon) on production?

Reply:
9.6 with no plans to change.